code/client/src/sdk/ue/game/traffic/c_race_manager.h

```diff
@@ -2,8 +2,12 @@
 
 namespace SDK {
     namespace ue::game::traffic {
+
+        /**
+         * Stubbed ingame Race Manager.
+         * For MP we avoid this, so just a stub
+         */
         class C_RaceManager {
```

Comment:
keep or remove? might be useful for future reference

Reply:
keep

Reply:
However, regarding the comment, i don't think we should have MP related comments here because in a long-term, sdk could be exported to other projects

Reply:
I've removed references to MP.